integration_test/docker/celestia-app/config/gentx/gentx-7a76f62d381128bc3f5a255b89d058046d923ace.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"tomek","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"celestia12e8tccsthuwfdfqv4dsdeh4kgmfrpnjhrem26a","validator_address":"celestiavaloper12e8tccsthuwfdfqv4dsdeh4kgmfrpnjhxxenvm","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"pGrjGKma3qFyt2vs7hIkzjMkd+ksJ4T5nmgqE3c30PI="},"value":{"denom":"celestia","amount":"5000000000"}}],"memo":"7a76f62d381128bc3f5a255b89d058046d923ace@192.168.1.110:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A88njG7WKYU2gYWJfu6DEWRNRg+qMRjTBBobgJK8nA9M"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["GDDpLaokeXrks1DfG9uPhHV4+9tPVQ7FOO6hWRv9RF8/A6qF7S7bxJmZHkF8yrZHmU4jzFnvBY1GIlOmLE9kWQ=="]}
```

Comment:
shouldn't this be generated on the fly instead of committed into the code here? Same for genesis.json, config.toml etc.

Reply:
Creating genesis file, as you know, is a bit complicated. I see no gain in 'randomly generated' genesis in this case. All we need is a minimal setup that creates blocks on fake/ephemeral/custom/temporary network and expose celestia-node API.

Reply:
I know. But I assume you've scripted this anyways? BTW @Bidon15 is also working on similar stuff for https://github.com/celestiaorg/test-infra/pull/23 
Maybe you can use the same script fwiw?